twenty-first/src/util_types/merkle_tree.rs

```diff
@@ -351,7 +340,7 @@ impl MerkleTree {
         // usize::MAX. Since the nodes are stored in a Vec, the number of nodes
         // can never exceed usize::MAX.
         // This proof by contradiction shows that unwrapping is fine.
-        let num_leafs = usize::try_from(self.num_leafs()).unwrap();
+        let num_leafs = self.num_leafs();
 
         self.nodes.iter().skip(num_leafs)
```

Comment:
The comment arguing that `.unwrap()` is fine is no longer necessary.

```suggestion
        self.nodes.iter().skip(self.num_leafs())
```

Reply:
Fixed in 4d7fc1718cdab39106900993d1155d15b6b49a21.